apps/cookbook/src/app/showcase/form-field-showcase/form-field-showcase.component.html

```diff
@@ -109,14 +109,26 @@ <h3>Date</h3>
     #inputDateExample
   ></cookbook-form-field-input-date-example>
 </cookbook-example-viewer>
+
+<p>
+  By default (or when
+  <code>[useNativeDatePicker]="false"</code>
+  is set explicitly) the value supplied to the input should be formatted according to the current
```

Comment:
Consider if it's relevant to specify this, as it doesn't read very well.
Maybe leave it out entirely or replace with something simpler — the second line already explains what happens when opting in to the native date picker.
```suggestion
  By default (i.e. when not using the native datepicker) the value supplied to the input should be formatted according to the current
```